pkg/programkind/programkind.go

```diff
@@ -131,6 +144,35 @@ func GetExt(path string) string {
 	return ext
 }
 
+var ErrUPXNotFound = errors.New("UPX executable not found in PATH")
+
+func ValidateUPX(path string) (bool, error) {
```

Comment:
Can you add a 1-line comment string? Based on the name I thought this validated the UPX installation, but this seems to validate that a program is UPX'd.

Reply:
Updated in [`f849ff6` (#731)](https://github.com/chainguard-dev/malcontent/pull/731/commits/f849ff659f7acd25b7a2f71e2b643ea13b027088#diff-98d64e984a50bac5a4a8314d16140f66b05e118db774dc53208d9896d61352f9).